exosphere-shared/example-apps/docker-service/application.yml

```diff
@@ -0,0 +1,11 @@
+name: docker-app
+description: An application that relies on dockerhub images
+version: 0.0.1
+
+services:
+  public:
+    local-service:
+      docker_image: exospheredev/local-service
+      location: ./local-service
```

Comment:
What happens if both the `docker_image` and `location` field are defined? Does it use only the local version of the service? If so shouldn't we only define one or the other?